cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java

```diff
@@ -967,7 +946,7 @@ public void getStreams(HttpRequest request, HttpResponder responder) {
   @Path("/streams/{stream-id}")
   public void getStreamSpecification(HttpRequest request, HttpResponder responder,
                                      @PathParam("stream-id") final String streamId) {
-    dataList(request, responder, Data.STREAM, streamId, null);
+    dataList(request, responder, store, dsFramework, Data.STREAM, streamId, null);
```

Comment:
Confirmed that this is not going to have a v3 equivalent, right? Make sure to add a note about documentation, if this was a public API.